CHANGELOG.md

```
@@ -1,7 +1,15 @@
-All notable changes to this project will be documented in this file.
+<!-- docs/changelog.md mirrors root CHANGELOG.md (the changelog-gate
+     keys on this file path). Keep both files in sync on every PR, or
+     move the gate to root. -->
 
 ## [Unreleased]
+
+### Added
+
 ### Added
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)